utility/aws.py

```diff
@@ -238,3 +242,35 @@ def detach_and_delete_volume(self, volume, timeout=120):
             "Delete response for volume: %s is: %s", ec2_volume.volume_id,
             delete_response
         )
+
+    def stop_node(self, instance_id):
```

Comment:
rename to stop_instance or stop_ec2_instance please

Reply:
done